[DBM] PR feedback on #15900 (#16016)
DataDog/documentation: layouts/shortcodes/dbm-mysql-agent-config-examples.md

    custom_queries:
    - query: SELECT age, salary, hours_worked, name FROM hr.employees;
      columns:
        - name: mysql.custom.employee_age
          type: gauge
        - name: mysql.custom.employee_salary
           type: gauge

>         - name: mysql.custom.employee_hours

Reviewer: For MySQL & SQL Server custom queries, we append to the prefix `<INTEGRATION>.` automatically so we'll probably want something like `custom.employee_age` or just `employee_age`.

           type: count
        - name: name
           type: tag
      tags:
        - 'table:employees'
```
### Working with hosts through a proxy

> If the Agent must connect through a proxy such as the [Cloud SQL Auth proxy](https://cloud.google.com/sql/docs/mysql/connect-admin-proxy), all telemetry will be tagged with the hostname of the proxy rather than the database instance. Utilize the `reported_hostname` option to set a custom override of the hostname detected by the agent.

Reviewer 2: ```suggestion
If the Agent must connect through a proxy such as the [Cloud SQL Auth proxy](https://cloud.google.com/sql/docs/mysql/connect-admin-proxy), all telemetry is tagged with the hostname of the proxy rather than the database instance. Utilize the `reported_hostname` option to set a custom override of the hostname detected by the Agent.
```
